packages/react-sdk/src/components/Layout/Layout.tsx

```diff
@@ -53,7 +53,12 @@ const TabPanelStyled = styled(TabPanel)(() => ({
   },
 }));
 
-export function Layout() {
+export type LayoutProps = {
+  // Height of component, default to '100vh'
+  height?: number | string;
```

Comment:
What does a number mean here? Should it not always include a unit?

Reply:
Could be just a string, was not sure about type, checked some mui code and saw number that would end up with pixels, so used `number | string` there.